File: VOLUME 2/BOTÃO INLINE/CLICOU NO BOTÃO.py
from telegram import InlineKeyboardButton, InlineKeyboardMarkup
from telegram.ext import Updater, CallbackQueryHandler

# Configuração do bot
bot_token = 'seu_token_aqui'
updater = Updater(bot_token, use_context=True)

# Função para lidar com a consulta do botão
def button_handler(update, context):
    query = update.callback_query
    query.answer()
    query.edit_message_text(text="Você clicou no botão!")

# Função para enviar a mensagem com o botão inline
def send_inline_button(update, context):
    keyboard = [[InlineKeyboardButton("Clique aqui", callback_data='button_clicked')]]
    reply_markup = InlineKeyboardMarkup(keyboard)
    update.message.reply_text('Clique no botão abaixo:', reply_markup=reply_markup)

# Registro dos handlers
updater.dispatcher.add_handler(CallbackQueryHandler(button_handler))
updater.dispatcher.add_handler(CommandHandler('start', send_inline_button))

# Inicialização do bot
updater.start_polling()
updater.idle()

File: VOLUME 2/BOTÃO INLINE/CLICOU NO BOTÃO.js
const TelegramBot = require('node-telegram-bot-api');

// Configuração do bot
const botToken = 'seu_token_aqui';
const bot = new TelegramBot(botToken, { polling: true });

// Função para lidar com a consulta do botão
bot.on('callback_query', (query) => {
  bot.answerCallbackQuery(query.id);
  bot.sendMessage(query.message.chat.id, 'Você clicou no botão!');
});

// Função para enviar a mensagem com o botão inline
bot.onText(/\/start/, (msg) => {
  const chatId = msg.chat.id;
  const keyboard = {
    inline_keyboard: [
      [{ text: 'Clique aqui', callback_data: 'button_clicked' }]
    ]
  };
  const message = 'Clique no botão abaixo:';
  bot.sendMessage(chatId, message, { reply_markup: keyboard });
});

